### ISSUE TITLE
the necessity of "metaSGD" instead of direct "torch.nn.SGD"

### ISSUE THREAD

**Issue by KeysaYoung** [ShiYunyi/Meta-Weight-Net_Code-Optimization#4]
Hi, thanks for your elegant code.
I haven't figured out the necessity of "metaSGD" instead of direct "torch.nn.SGD", either. Is it possible that you provide more detailed descriptions? Thanks a lot!

**Comment by limengyang1992** [comment]
I also have the same question. Have you solved it? I tried to replace SGD and found that the accuracy is almost the same

![image](https://user-images.githubusercontent.com/73928007/161046091-f9cc6c7e-3c4f-4501-a42e-fdd5006a4a92.png)

**Comment by WANGBohaO-jpg** [comment]
the same question

**Comment by YF-SA** [comment]
你好使用内置的优化器可以吗？为什么重新书写优化器？
